src/main/java/com/dev/cinema/service/impl/UserServiceImpl.java

```diff
@@ -34,4 +34,5 @@ public User add(User user) {
     public User getById(Long id) {
         return userDao.getById(id);
     }
+
```

Comment:
why add this empty line? 
it's not needed

Reply:
done